pubspec.lock

```diff
@@ -306,6 +306,6 @@ packages:
       name: yaml
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.1.10"
+    version: "2.1.12"
 sdks:
-  dart: ">=1.17.0-dev.6.2 <2.0.0"
+  dart: ">=1.21.0-dev.2.0 <2.0.0"
```

Comment:
This is a generated file.  How can I make sure pub doesn't undo this if I force-override it manually?